src/app/fdctl/run/tiles/fd_verify.h

```diff
@@ -40,6 +40,8 @@ typedef struct {
   ulong       out_chunk;
 } fd_verify_ctx_t;
 
+#define FD_VERIFY_DEDUP_TAG_FROM_PAYLOAD_SIG(payload_sig_p) FD_LOAD( ulong, (payload_sig_p) )
```

Comment:
nit but i don't really get the point of this macro, the inline FD_LOAD is clearer to me

Reply:
I thought this would make it easier for search and replace if the part of a signature buffer that's used for deduplication ever gets extended or changed, now that both fd_verify.c and fd_dedup.c are extracting the dedup tag.  Happy to revert this.  Let me know.